sdk/iotcentral/arm-iotcentral/README.md

```diff
@@ -1,93 +1,87 @@
 ## Azure IotCentralClient SDK for JavaScript
 
-This package contains an isomorphic SDK for IotCentralClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for IotCentralClient.
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
+- Node.js version 8.x.x or higher
 - Browser JavaScript
 
-### How to Install
+### How to install
 
+To use this SDK in your project, you will need to install two packages.
+- `@azure/arm-iotcentral` that contains the client.
+- `@azure/identity` that contains different credentials for you to authenticate the client using Azure Active Directory.
+Install both packages using the below commands.
```

Comment:
Looks like we need a line break before this line, otherwise it appears as a single line in the rendered markdown